src/openrct2-ui/windows/Dropdown.cpp

```diff
@@ -457,9 +457,14 @@ void window_dropdown_show_colour(rct_window* w, rct_widget* widget, uint8_t drop
     // Show dropdown
     window_dropdown_show_image(
         w->windowPos.x + widget->left, w->windowPos.y + widget->top, widget->bottom - widget->top + 1, dropdownColour,
-        DROPDOWN_FLAG_STAY_OPEN, COLOUR_COUNT, 12, 12, gAppropriateImageDropdownItemsPerRow[COLOUR_COUNT]);
+        DROPDOWN_FLAG_STAY_OPEN, COLOUR_COUNT, 12, 12, _appropriateImageDropdownItemsPerRow[COLOUR_COUNT]);
 
     gDropdownIsColour = true;
     gDropdownLastColourHover = -1;
     gDropdownDefaultIndex = defaultIndex;
 }
+
+uint32_t dropdown_get_appropriate_image_dropdown_items_per_row(uint32_t numItems)
```

Comment:
Should probably use CamelCase for this new function. Although might not make too much sense since it would be out of place in this file.

Reply:
I don't think we have any C-style function that uses CamelCase - only C++ methods seem to use it.

Reply:
There are a good bunch scattered throughout the code base. The coding style which we (sometimes) follow assumes all functions are CamelCase.